### PR TITLE
feat(timeline): show attendance state by adding colored circle around…

### PR DESCRIPTION
**Pull request by huerlisi**
… avatars

### CODE REVIEW

**Comment by huerlisi** [comment]
android-bottom-bar1

File: app/assets/stylesheets/attendance.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
+$attendance-state-color-invited: $color-gray-dark;
+$attendance-state-color-attending: $color-green-dark;
+$attendance-state-color-declined: $color-red-dark;
+
+img.circle.attendance-state-invited {
```
Comment (koffeinfrei) [inline app/assets/stylesheets/attendance.scss]: It's not really necessary to bind this specifically to `img`.

**Comment by koffeinfrei** [comment]
Maybe sort by attendance (green first, ...)?

**Comment by koffeinfrei** [comment]
:+1:

**Pull request merged**